django/db/migrations/operations/fields.py

```diff
@@ -154,11 +154,17 @@ def reduce(self, operation, app_label):
 class RemoveField(FieldOperation):
```

Comment:
We need to update the documentation for this field: https://docs.djangoproject.com/en/5.0/ref/migration-operations/#removefield and add a release note.

Reply:
Done!